spyder/plugins/ipythonconsole/utils/client.py

```diff
@@ -102,14 +102,14 @@ def tunnel_to_kernel(
             )
         elif sshkey is not None:
             self.__tunnel_handler = KernelClientTunneler.new_connection(
-                tunnel=hostname,
                 password=password,
                 client_keys=[sshkey],
+                **self._split_shh_address(hostname),
             )
-        else:
+        else:            
```

Comment:
```suggestion
        else:
```

Remove unnecessary blank spaces.